WordPress/src/main/java/org/wordpress/android/ui/photopicker/mediapicker/DeviceListBuilder.kt

```diff
@@ -0,0 +1,81 @@
+package org.wordpress.android.ui.photopicker.mediapicker
```

Comment:
hmm, that's a good point. On one hand having both in the same parent package makes it clear they are related to each other. On the other hand you're right that it would make it easier to remove the old code once this work is done. What do you think if I moved the new classes in another PR since there are already some other PRs that depend on this package location? (we can move the `PermissionsHandler` once we remove the photo picker 🤷 )

Reply:
SGTM 👍